onnxruntime/contrib_ops/cuda/bert/attention.cc

```diff
@@ -1,10 +1,9 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-#include "attention.h"
-#include "core/framework/tensorprotoutils.h"
 #include "core/providers/cuda/cuda_common.h"
 #include "core/providers/cuda/shared_inc/fpgeneric.h"
+#include "attention.h"
```

Comment:
can the related header include be first?
[https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)

Reply:
That one could happen to be included first. For others they needed to have the provider.api file first to define SHARED_PROVIDER.

Fixed